packages/cpp-debug/src/browser/editable-widget/memory-editable-table-widget.tsx

```diff
@@ -185,23 +184,26 @@ export class MemoryEditableTableWidget extends MemoryTableWidget {
     }
 
     protected submitMemoryEdits = async (): Promise<void> => {
-        // eslint-disable-next-line @typescript-eslint/no-unused-vars
-        for await (const _ of this.submitMemoryEditInOrder()) {
-            // Do nothing, but without lint errors.
+        for (const edit of this.submitMemoryEditInOrder()) {
+            try {
+                await this.memoryProvider.writeMemory(edit);
+            } catch (e) {
+                console.log('Problem writing memory with arguments', edit, '\n', e);
+            }
         }
     };
 
-    private * submitMemoryEditInOrder(): IterableIterator<Promise<DebugProtocol.WriteMemoryResponse | undefined>> {
+    private * submitMemoryEditInOrder(): IterableIterator<DebugProtocol.WriteMemoryArguments> {
         this.memoryEditsCompleted = new Deferred();
-        const addressesSubmitted = new Set<Long>();
+        const addressesSubmitted = new Set<string>();
         for (const k of this.pendingMemoryEdits.keys()) {
             const address = Long.fromString(k);
             const { address: addressToSend, value: valueToSend } = this.composeByte(address, true);
-            if (!addressesSubmitted.has(addressToSend)) {
-                const data = Array64.encode(new Uint8Array([parseInt(valueToSend, 16)]));
-                const writeMemoryArguments = { memoryReference: addressToSend.toString(), data };
-                yield this.memoryProvider.writeMemory?.(writeMemoryArguments) ?? Promise.resolve(undefined);
-                addressesSubmitted.add(addressToSend);
+            const memoryReference = '0x' + addressToSend.toString(16);
+            if (!addressesSubmitted.has(memoryReference)) {
+                const data = Buffer.from(valueToSend, 'hex').toString('base64');
```

Comment:
Also a minor thing, but not using `Array64` anymore leaves an unused import which makes `yarn` fail.

Reply:
Oops. Fixed.